### ISSUE TITLE
'mindboard' import statements

### ISSUE THREAD

**Issue by jmhansen** [fokoenecke/html_docx#6]
@fokoenecke, can you share some context about `mindboard` used in the following import statements?  
- `from mindboard.helpers.docx.html.tag_dispatchers import TagDispatcher`
- `from mindboard.helpers.docx.html.converter import DocxBuilder`

I am getting an import error and can't find anything about this library online, so am assuming it is something you are using locally.

**Comment by fokoenecke** [comment]
Yep, you are right. Its the project I wrote the code for. I never wanted to make it public in the first place. Since the project got stopped I never had the chance to clean this part or put it into Python-docx directly. 
I discovered some more "internal" stuff, like coded header formats. Additionally i have not had a look at Python-docx for quite some time. 
If I were you, I would have a look at my code and implement it yourself based on that.

**Comment by jmhansen** [comment]
Thanks @fokoenecke, I figured out another solution.

**Comment by tooh** [comment]
@nesnahnoj  Can you share the solution you found ?

**Comment by mammique** [comment]
Yes, please, what solution? Thanks.

**Comment by mammique** [comment]
Type this in the html_docx directory: `find -name "*.py" | xargs sed s/mindboard.helpers.docx.//g -i`
